source/geod24/LocalRest.d

```diff
@@ -1619,3 +1634,59 @@ unittest
     node_1.ctrl.shutdown();
     thread_joinAll();
 }
+
+/// Example of a custom (de)serialization policy
+unittest
+{
+    static struct Serialize
+    {
+    static:
+        public immutable(ubyte[]) serialize (T) (auto ref T value) @trusted
```

Comment:
Ah I see, lvalues will work but only if `serialize` is a template with `auto ref`.